### ISSUE TITLE
Accessibility improvement suggestions

### ISSUE THREAD

**Issue by adamsamec** [jsbin/jsbin#3490]
Plese, could you improve the accessibility of JSBin for screen reader users? My concrete both most fundamental  and easy-to-implement suggestions for improvements are the following:

* Add a heading before each of the editor text fields, like this: &lt;h2&gt;HTML&lt;/h2&gt;. This will significantly help screen reader users to navigate to the desired location on the page via the jump to next/previous heading feature of the screen reader. The headings should be introduced for all the important regions of the page, i.e., the HTML, CSS, JavaScript and Output regions.
* Add the "aria-label" attribute for all the editor text fields, like this: &lt;textarea aria-label="HTML"&gt; .
